src/atoms/social/speaker/presentational.js

```diff
@@ -27,25 +28,29 @@ const socialItemStyle = {
   alignItems: 'center'
 };
 
+const SocialItem = styled.div`
+  ${socialItemStyle}
+`;
+
 export default function SpeakerSocialIcons({twitter, github}) {
   const nonHoveredIconColor = '#888';
   const TwitterIcon = hovered => (
     <li css={listItemStyles}>
       <ExternalLink to={`https://twitter.com/${twitter}`}>
-        <div css={socialItemStyle}>
+        <SocialItem>
```

Comment:
the type of thing that i was thinking was that the `<ExternalLink>` would have a variant that knows to render an icon next to text and so passing something like `variant="inlineIconWithText"` to `<ExternalLink>` would allow it to apply the styles directly inside the `<ExternalLink>` (instead of extra elements here or the parent passing actual style rules to `<ExternalLink>`. does what i'm suggesting make sense? still open to disagreement about that being the right approach, but trying to make sure i'm at least communicating what i mean.

Reply:
Perhaps the overall idea makes sense, but I'm not sure how to implement this atm. I need to do some more research on how variants work.. this fits in to the approach with [design system](https://github.com/dsmjs/components/issues/88)?

Reply:
the variant approach is not much more than a technique to enable encapsulating the style rules in the component but still allowing a parent to specify a variation from a limited list, so not necessarily tied to anything higher level, like a design system. it just better aligns with the philosophy that the component knows how to style itself, regardless of context, instead of letting that control bleed out to random parents.

as far as implementation goes, that is totally up to us. like the internal-link, it can start as a short conditional, but might grow into a more complex case-statement or similar over time. in the case of the external-link, the variants that come to mind so far are just `textOnly` and `inlineIconWithText`. i think `textOnly` makes sense to be the default when a variant isnt explicitly passed by a parent.

youre more than welcome to finish this out, but if you'd rather i take a stab at exposing the variants from the external-link component, just let me know.

Reply:
Yeah feel free to take a stab at it! I can close this PR for now. I'll try to keep an eye out for how you want to do it

Reply:
works for me. ill try to incorporate your changes along with that and ill send you the diff at that point